app/assets/javascripts/components/top_panel.js.jsx

```diff
@@ -51,13 +62,24 @@ var TopPanel = React.createClass({
         </div>
         <div>
           <div className='input-field form-input'>
-            <input onChange={this.updateValue} type='number' step='any' name='expense' value={this.state.value} />
-            <label htmlFor='expense'>{'Add ' + this.state.currentDisplay + ' spending'} </label>
+            <input onChange={this.updateValue} type='number' step='any' name='expense' value={this.state.value} placeholder={"Add " + this.state.currentDisplay + " spending"} />
           </div>
-          <div className='submit-btn btn deep-orange' onClick={this.submitForm}>
+          <div className='submit-btn btn deep-orange' onClick={this.submitForm} >
             Add
           </div>
         </div>
+        <div>
+          <div className={'next-week-check ' + this.state.currentDisplay + "-check"}>
+            <input
+              type="checkbox"
+              id="next_week"
+              className="next_week"
```

Comment:
lets remove the ID unless we need it, and class/id names should use `-` instead of `_` per convention